account_invoice_hide_payment_button/__manifest__.py

```diff
@@ -1,17 +1,15 @@
-# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
-# Eric Antones <eantones@nuobit.com>
+# Copyright NuoBiT - Eric Antones <eantones@nuobit.com>
+# Copyright NuoBiT 2025 - Bijaya Kumal <bkumal@nuobit.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
 {
     "name": "Invoice hide payment button",
     "summary": "This module hides payment button on invoices",
-    "version": "14.0.1.0.0",
+    "version": "16.0.1.0.0",
     "category": "Accounting",
-    "author": "NuoBiT Solutions, S.L., Eric Antones",
+    "author": "NuoBiT Solutions SL, Eric Antones",
```

Comment:
```suggestion
    "author": "NuoBiT Solutions SL",
```
This is deprecated, remove always the name from all the manifests from now on.